jobs/base_model_validator.py

```diff
@@ -0,0 +1,161 @@
+# coding: utf-8
+#
+# Copyright 2021 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Beam PTransform and DoFn for base model validator functions"""
```

Comment:
Please add one line above this and add a period at the end.

Also could you write this as more of a fileoverview that explains in layperson terms what these are / what goes in this file? The current description is hard to understand for a contributor who is not familiar with the terminology. It is OK if the resulting fileoverview is longer than one line.

Reply:
Done.